ios-sdk/TangramVC.swift

```diff
@@ -12,6 +12,7 @@ class TangramVC:  TGMapViewController{
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        self.queueSceneUpdate("sources.mapzen.url_params.api_key", withValue: "vector-tiles-7bfyfz8")
```

Comment:
Two things:
- I don't see any scene loaded anywhere?
- `self.applySceneUpdate()` must be called somewhere for this update to be effective.

For future reference, you can see here what is being triggered once a scene gets loaded: https://github.com/tangrams/tangram-es/blob/master/core/src/tangram.cpp#L182, most important thing to notice is that scene updates get cleared up at any `loadScene*` call.